src/Components/Header/Header.jsx

```diff
@@ -194,48 +192,55 @@ function Header({ program, loadProgram, history, admin, curRoute, routes }) {
         <RightSideHeaderWrapper>
           {appUser.role === "Admin" || hasAdminAccessForCurrentProgram ? (
             <RightSideHeaderWrapper>
-              <p> {curRoute.title} </p>
-              <ArrowDropDownCircleOutlinedIcon
-                style={{ marginLeft: "4px", margin: "12px" }}
-                onClick={handleClickAdminMenu}
-              ></ArrowDropDownCircleOutlinedIcon>
-              <Menu
-                elevation={0}
-                id="simple-menu"
-                anchorEl={adminMenuAnchor}
-                keepMounted
-                open={Boolean(adminMenuAnchor)}
-                onClose={() => {
-                  setAdminMenuAnchor(null);
-                }}
-                style={{ marginTop: HEADER_HEIGHT / 2 }}
-              >
-                {routes != null ? (
-                  <div style={{ border: "1px solid #ccc" }}>
-                    {routes.map((route, index) => {
-                      if (route.title !== curRoute.title) {
-                        return (
-                          <MenuItem
-                            key={index}
-                            onClick={() => history.push(route.path)}
-                            visible={(
-                              route.title !== curRoute.title
-                            ).toString()}
-                          >
-                            {route.title}
+              <p style={{ marginLeft: "4px", margin: "12px" }}>
+                {" "}
+                {curRoute.title}{" "}
+              </p>
+              {routes.length > 0 ? (
```

Comment:
These changes are for displaying the dropdown or not if there are no routes to show.